packages/server/api/src/app/workers/queue/throttled-job-queue.ts

```diff
@@ -0,0 +1,107 @@
+import {
+    AppSystemProp,
+    getThrottledJobsQueueName,
+    memoryLock,
+    QueueName
+} from '@activepieces/server-shared'
+import { isNil } from '@activepieces/shared'
+import { Queue } from 'bullmq'
+import { BullMQOtel } from 'bullmq-otel'
+import { FastifyBaseLogger } from 'fastify'
+import { redisConnections } from '../../database/redis-connections'
+import { dedicatedWorkers } from '../../ee/platform/platform-plan/platform-dedicated-workers'
+import { system } from '../../helper/system/system'
+import { AddJobParams, JobType } from './queue-manager'
+
+const dedicatedWorkersQueues = new Map<string, Queue>()
+
+export const throttledJobQueue = (log: FastifyBaseLogger) => ({
```

Comment:
The dedicated queue is a feature we sell for the enterprise on the cloud, where we completely separate their queue and workers that consume the jobs.


Given that, we don't need a throttle queue for each platform and rather one queue.

I think this file should be deleted and replaced with the following inside the worker rate limiter file.

```ts
   const isOtpEnabled = system.getBoolean(AppSystemProp.OTEL_ENABLED)
            const throttledQueue = new Queue("throttledJobs", {
                telemetry: isOtpEnabled ? new BullMQOtel(queueName) : undefined,
                connection: await redisConnections.create(),
            })
```

And it can be used directly to add.